test/unit-tests/intercept.test.js

```diff
@@ -1,161 +0,0 @@
-const expect = require('chai').expect;
```

Comment:
again, making sure that this deletion is intentional

Reply:
this was not intentional.. removed accidentally will be adding it back